BPOR/BPOR.Rms/Views/Email/Partials/_TemplateSelect.cshtml

```diff
@@ -16,7 +16,7 @@
     }
 </style>
 
-<div class="govuk-form-group @(Html.ViewData.ModelState["SelectedTemplateId"]?.Errors.Count > 0 ? "govuk-form-group--error" : "")" asp-for="SelectedTemplateId">
+<div class="govuk-form-group" @(Html.ViewData.ModelState["SelectedTemplateId"]?.Errors.Count > 0 ? "govuk-form-group--error" : "") asp-for="SelectedTemplateId">
```

Comment:
This doesn't look correct. `govuk-form-group--error` should be added as a class.